docs/index.rst

```diff
@@ -419,6 +419,22 @@ Information will be lost but it allows to control any potential performance impa
     tracer.sampler = RateSampler(sample_rate)
 
 
+Resolving deprecation warnings
```

Comment:
Section that is in line with Django Web Framework. Credits to this great community!

Ref: https://docs.djangoproject.com/en/2.0/howto/upgrade-version/#resolving-deprecation-warnings